easel/lib/screens/creator_hub/creator_hub_screen.dart

```diff
@@ -168,49 +163,63 @@ class _CreatorHubContentState extends State<CreatorHubContent> {
-                  child: AutoSizeText(
-                    LocaleKeys.welcome_msg.tr(),
-                    textAlign: TextAlign.start,
-                    maxLines: 4,
-                    style: titleStyle.copyWith(fontSize: isTablet ? 9.sp : 12.sp),
-                  ),
+                  child: getDescriptionWidget(isDraftCreated: viewModel.nftDraftList.isNotEmpty || viewModel.nftPublishedList.isNotEmpty),
                 ),
-                SizedBox(height: 32.h),
+                SizedBox(height: 15.h),
                 Padding(
                   padding: EdgeInsets.symmetric(horizontal: 20.w),
                   child: Row(
                     children: [
-                      buildOutlinedBox(title: LocaleKeys.draft.tr(), viewModel: viewModel, collectionType: CollectionType.draft),
+                      if (viewModel.selectedCollectionType == CollectionType.draft)
+                        buildSelectedBox(
+                          title: LocaleKeys.draft.tr(),
+                          viewModel: viewModel,
+                          color: EaselAppTheme.kLightRed,
+                          collectionType: CollectionType.draft,
+                        )
+                      else
+                        buildOutlinedBox(
+                          title: LocaleKeys.draft.tr(),
+                          viewModel: viewModel,
+                          collectionType: CollectionType.draft,
+                        ),
                       SizedBox(width: 14.w),
-                      buildOutlinedBox(title: LocaleKeys.published.tr(), viewModel: viewModel, collectionType: CollectionType.published),
+                      if (viewModel.selectedCollectionType == CollectionType.published)
+                        buildSelectedBox(
+                          title: LocaleKeys.published.tr(),
+                          viewModel: viewModel,
+                          color: EaselAppTheme.kDarkGreen,
+                          collectionType: CollectionType.published,
+                        )
+                      else
+                        buildOutlinedBox(
+                          title: LocaleKeys.published.tr(),
+                          viewModel: viewModel,
+                          collectionType: CollectionType.published,
+                        ),
                       const Expanded(child: SizedBox()),
                       SizedBox(width: 16.w),
                       InkWell(
                         onTap: () => viewModel.updateViewType(ViewType.viewGrid),
-                        child: SvgPicture.asset(SVGUtils.kGridIcon, height: 15.h, color: viewModel.viewType == ViewType.viewGrid ? EaselAppTheme.kBlack : EaselAppTheme.kGreyIcon),
+                        child: SvgPicture.asset(
+                          SVGUtils.kGridIcon,
+                          height: 18.h,
+                          color: getButtonColor(isSelected: viewModel.viewType == ViewType.viewGrid),
```

Comment:
What viewType has to do with color?

Reply:
if the selected viewType is a list then the color of the list view icon will be black and the color of the grid view icon will be grey and wise versa.


<img width="280" alt="image" src="https://user-images.githubusercontent.com/115459890/204502140-a7f5076b-75e8-4354-883c-336567199060.png">